packages/core/docs/changelog/6035.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  description: 'Remove generate command from CT and BP themes',
+  link: 'https://github.com/vuestorefront/vue-storefront/pull/6035',
+  isBreaking: false,
+  breakingChanges: [],
+  author: 'lukaszjedrasik',
+  linkToGitHubAccount: 'https://github.com/lukaszjedrasik'
+};
```

Comment:
Please create two changelogs - one in `docs/changelog`, other in `docs/commercetools/changelog`. Both should have `description`:

``Remove `generate` command from `package.json` ``